projects/google-cloud/compute-engine.tf

```diff
@@ -0,0 +1,29 @@
+resource "google_compute_instance" "ocf-offsite" {
```

Comment:
We're running a VM offsite? What is this for?

Reply:
It was going to be for external monitoring, but we never got around to using it and the actual monitoring checks are probably better than setting up a VM to do it. Anyway, it's in the free tier and has just been running for the last few years without really doing anything, so if you have good ideas for things to do with a small off-site host then this might work well.